float32 battery_percentage
geometry_msgs/Vector3 position
geometry_msgs/Vector3 velocity

